lib/stages/frbPostProcess.cpp

```diff
@@ -1,7 +1,3 @@
-// TODO Where do these live?
```

Comment:
I can't figure out which `#include` frbPostProcess gets the defines from?

Reply:
`frbPostProcess` doesn't actually use `[un]likely` so it just doesn't need it.

Reply:
Hah, you're right! I remembered its use from pulsarPostProcess, and never noticed that it's not used in frbPostProcess even though the two are so similar. It probably should be, to mark the `if (startup)` case in line 48, but that's a different issue. :-)